cmd/katalyst-agent/app/options/sysadvisor/poweraware/power_aware_plugin.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2022 The Katalyst Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package poweraware
+
+import (
+	cliflag "k8s.io/component-base/cli/flag"
+
+	"github.com/kubewharf/katalyst-core/pkg/config/agent/sysadvisor/poweraware"
+)
+
+type PowerAwarePluginOptions struct {
+	disabled                  bool
```

Comment:
good point - since there is already SysAdvisorPlugins mechanism to exclude(disable) pap plugin, no need to have its disable field. will change code to reflect such reduction.